case-studies/TvsA.md

```diff
@@ -0,0 +1,45 @@
+---
+description: As part of the Tech vs Abuse programme coordinated by CAST & Comic Relief, Founders and Coders proposed an intensive product management programme in which we'd validate this curriculum. The programme was not the best place to validate this methodology and curriculum and so we pivoted to deliver this in a dedicated programme, the Collaborative Digital Training.
+---
+
+# Case Study: Tech vs Abuse
+
+## Introduction
+
+Our proposed engagement on Tech vs Abuse was to deliver an intensive training programme in Discovery, Definition, Prototyping and Product Management. The delivery of this programme was intended to be the validation of this curriculum. What we didn't know when planning this programme was the intesity of the broader Tech vs Abuse programme. Fitting our engagement into this programme proved a bridge too far and so we pivoted to validate this curriculum on a smaller scale as a dedicated programme, the [Collbaorative Digital Training](./CDT.md).
+
+---
+
+## Structure
+
+### Discovery
+
+- Five Why's
+- The Bigger Picture
+- Problem Statements
+- Assumptions & Validators
+- User Personas
+- User Research Scripting
+
+### Definition
+
+- Idenitfying Key Insights
+- Inspiration
+- Sketching
+- User Journey and User Stories
+
+### Build
+
+- Managing a sprint
+- Prioritising a backlog
+- The Product Owner Role
+- Sprint Reviews
+
+---
+
+## Reflections
+
+- We learnt a lot from this enagagement and it was the foundation of this curriculum. Crucially, we found that within the charity sector upskilling those who will lead on digital products is an urgent need. For that purpose, we continue to deliver training programmes and maintain this curriculum.
+- Interventions should be responsive to the capacity of organisations. An organisation may already be overstretched and when this is the case, our work must adapt to their capacity. For this reason, we recommend that the core exercises are completed in workshops and additional time and support is given to those organisations who do have more time to engage.
+- Expectations should be managed ahead of programmes beginning. For this reason, we give clear programme outlines as far in advance as reasonable and set clear expectations from the outset. For example, when asking organisations to conduct user research we'd communicate this ahead of time to give organisations appropriate time to coordinate with their users. Furthermore, we are responsive to where this may not be possible (usually due to safeguarding concerns) and adjust our outlines to accomodate this - for example, on a learning programme, conducting research with those _close to users_ is appropriate (i.e. parents for a children's charity).
```

Comment:
```suggestion
- Expectations should be managed ahead of programmes beginning. For this reason, we give clear programme outlines as far in advance as reasonable and set clear expectations from the outset. For example, when asking organisations to conduct user research we'd communicate this ahead of time to give organisations appropriate time to coordinate with their users. Furthermore, we are responsive to where this may not be possible (usually due to safeguarding concerns) and adjust our outlines to accommodate this - for example, on a learning programme, conducting research with those _close to users_ is appropriate (i.e. parents for a children's charity). This can also cause organisations to reflect on who their services are for and who exactly to engage. 
```